### PR TITLE
Archetype for new practice page

### PR DESCRIPTION
**Pull request by joemoore**
- Archetype file
- Updated Make file with commend to create a new practice
- Updated README

### CODE REVIEW

File: archetypes/practices.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: true
+
+title: "{{ (replace .Name "-" " ") | title }} - Page Title"
+linkTitle: "{{ replace (replace .Name "what-is-" "") "-" " " | title }} - Will Appear in Links"
```
Comment (zmb3) [inline archetypes/practices.md:6]: What's this `what-is` bit?

Comment (joemoore) [reply archetypes/practices.md:6]: Good catch

File: archetypes/practices.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: true
+
+title: "{{ (replace .Name "-" " ") | title }} - Page Title"
+linkTitle: "{{ replace (replace .Name "-" "") "-" " " | title }} - Will Appear in Links"
```
Comment (bryanfriedman) [inline archetypes/practices.md:6]:
```suggestion
linkTitle: "{{ replace .Name "-" " " | title }} - Will Appear in Links"
```

Comment (bryanfriedman) [reply archetypes/practices.md:6]: I think this is what you ultimately want.

**Review by bryanfriedman — COMMENTED** [review]
Other than the one small suggested change, this LGTM. I say :shipit:!

**Pull request merged**